<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BackLoan</class>
 <widget class="QMainWindow" name="BackLoan">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>336</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>960</width>
    <height>336</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>960</width>
    <height>336</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Raleway</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Devolución del prestamo</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/book-flat.ico</normaloff>:/book-flat.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="gbMain">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>941</width>
      <height>241</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(238, 244, 242); color: rgb(35, 54, 86);</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QGroupBox" name="gbInformation">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>921</width>
       <height>151</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Raleway</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Información del prestamo</string>
     </property>
     <widget class="QLabel" name="lbStudentCode">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Código del alumno</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lneStudentCode">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>28</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255,255,255);</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbName">
      <property name="geometry">
       <rect>
        <x>247</x>
        <y>30</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Nombre del alumno</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lneName">
      <property name="geometry">
       <rect>
        <x>377</x>
        <y>28</y>
        <width>291</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255,255,255);</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbCellphone">
      <property name="geometry">
       <rect>
        <x>677</x>
        <y>32</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Telefono</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lneCellphone">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>30</y>
        <width>171</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255,255,255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="lbExistence">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>62</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Ejemplar</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbTitle">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>62</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Titulo</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbISBN">
      <property name="geometry">
       <rect>
        <x>135</x>
        <y>62</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>ISBN</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbSecondDate">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>122</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Fecha de fin</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbGenero">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>92</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Genero</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbFirstDate">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>122</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Fecha de inicio</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbFormate">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>92</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Formato</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbPublisher">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>90</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Editorial</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lneExistence">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>60</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255,255,255);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lneISBN">
      <property name="geometry">
       <rect>
        <x>173</x>
        <y>60</y>
        <width>191</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255,255,255);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lneTitle">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>60</y>
        <width>501</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255,255,255);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lnePublisher">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>88</y>
        <width>211</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255,255,255);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lneGenero">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>90</y>
        <width>221</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255,255,255);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lneFormate">
      <property name="geometry">
       <rect>
        <x>355</x>
        <y>90</y>
        <width>271</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255,255,255);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lneFirstDate">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>120</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255,255,255);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lneSecondDate">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>120</y>
        <width>145</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Raleway</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255,255,255);</string>
      </property>
     </widget>
     <zorder>lbStudentCode</zorder>
     <zorder>lneStudentCode</zorder>
     <zorder>lbName</zorder>
     <zorder>lneName</zorder>
     <zorder>lbCellphone</zorder>
     <zorder>lneCellphone</zorder>
     <zorder>lbExistence</zorder>
     <zorder>lbTitle</zorder>
     <zorder>lbISBN</zorder>
     <zorder>lbSecondDate</zorder>
     <zorder>lbGenero</zorder>
     <zorder>lbFirstDate</zorder>
     <zorder>lbFormate</zorder>
     <zorder>lbPublisher</zorder>
     <zorder>lneExistence</zorder>
     <zorder>lneISBN</zorder>
     <zorder>lneTitle</zorder>
     <zorder>lnePublisher</zorder>
     <zorder>lneFirstDate</zorder>
     <zorder>lneGenero</zorder>
     <zorder>lneFormate</zorder>
     <zorder>lneSecondDate</zorder>
    </widget>
    <widget class="QPushButton" name="pbAcept">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>231</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Raleway</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#pbAcept {
background-color: rgb(123, 155, 166);
color:  rgb(255, 255, 255);
border-radius: 5px;
 }

 QPushButton#pbAcept:hover {
background-color: rgb(65, 91, 118);
color:  rgb(255, 255, 255);
 }</string>
     </property>
     <property name="text">
      <string>Aceptar devolución</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbExit">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>170</y>
       <width>231</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Raleway</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#pbExit {
background-color: rgb(123, 155, 166);
color:  rgb(255, 255, 255);
border-radius: 5px;
 }

 QPushButton#pbExit:hover {
background-color: rgb(65, 91, 118);
color:  rgb(255, 255, 255);
 }</string>
     </property>
     <property name="text">
      <string>   Salir</string>
     </property>
     <property name="icon">
      <iconset resource="images.qrc">
       <normaloff>:/door_out.ico</normaloff>:/door_out.ico</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbId">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>941</width>
      <height>52</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(238, 244, 242); color: rgb(35, 54, 86);</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lbId">
       <property name="font">
        <font>
         <family>Raleway</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>ID DEL PRESTAMO</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lneIdLoan">
       <property name="font">
        <font>
         <family>Raleway</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="maxLength">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbSearch">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Raleway</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#pbSearch {
background-color: rgb(123, 155, 166);
color:  rgb(255, 255, 255);
border-radius: 5px;
 }

 QPushButton#pbSearch:hover {
background-color: rgb(65, 91, 118);
color:  rgb(255, 255, 255);
 }</string>
       </property>
       <property name="text">
        <string>Buscar</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>-6</x>
      <y>-5</y>
      <width>971</width>
      <height>351</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(65, 91, 118);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>gbMain</zorder>
   <zorder>gbId</zorder>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pbExit</sender>
   <signal>clicked()</signal>
   <receiver>BackLoan</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>800</x>
     <y>267</y>
    </hint>
    <hint type="destinationlabel">
     <x>539</x>
     <y>324</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
